blueocean-web/src/main/js/DevelopmentFooter.jsx

```diff
@@ -5,8 +5,13 @@ import revisionInfo from '../../../target/classes/io/jenkins/blueocean/revisionI
 
 export class DevelopmentFooter extends Component {
     render() {
-        if (!revisionInfo.name) {
-            return null;
+        if (!revisionInfo || !revisionInfo.name) {
+            var blueOceanVersion = document.getElementsByTagName('head')[0].getAttribute('data-blue-ocean-version');
+            return (
+                <div className="development-footer">
+                    <span>Blue Ocean UI v{blueOceanVersion}</span>
```

Comment:
`{blueOceanVersion}` will potentially be '1.0' or '1.1', etc., it would be too barren without some additional text...